### PR TITLE
feat: Expose a flat API controller from neuron wallet to UI layer

### PR DESCRIPTION
**Pull request by ashchan**
Neuron UI should require this api controller only.

### CODE REVIEW

File: packages/neuron-ui/src/services/remote/apiMethodWrapper.ts [hunk 1]
```diff
@@ -42,13 +35,18 @@ export const controllerMethodWrapper = (controllerName: string) => (
   if (!window.remote) {
     return RemoteNotLoadError
   }
-  const controller = window.remote.require(`./controllers/${controllerName}`).default
+  const controller = window.remote.require('./controllers/api').default
   if (!controller) {
```
Comment (Keith-CY) [inline packages/neuron-ui/src/services/remote/apiMethodWrapper.ts]: The validation should include the error of `./controllers/api is not found`

File: packages/neuron-ui/src/services/remote/apiMethodWrapper.ts [hunk 2]
```diff
@@ -75,6 +73,5 @@ export const controllerMethodWrapper = (controllerName: string) => (
 
 export default {
   RemoteNotLoadError,
-  controllerNotLoaded,
-  controllerMethodWrapper,
+  controllerMethodWrapper: apiMethodWrapper,
```
Comment (Keith-CY) [inline packages/neuron-ui/src/services/remote/apiMethodWrapper.ts]: `controllerMethodWrapper` should be renamed `apiMethodWrapper`

File: packages/neuron-ui/src/services/remote/apiMethodWrapper.ts [hunk 3]
```diff
@@ -21,14 +21,7 @@ export const RemoteNotLoadError = {
...
 
-export const controllerNotLoaded = (controllerName: string) => ({
-  status: 0 as 0,
-  message: {
-    content: `${controllerName} controller not loaded`,
-  },
-})
-
-export const controllerMethodWrapper = (controllerName: string) => (
+export const apiMethodWrapper = () => (
```
Comment (Keith-CY) [inline packages/neuron-ui/src/services/remote/apiMethodWrapper.ts]: The most outside wrapper is redundant now.

File: packages/neuron-ui/src/services/remote/apiMethodWrapper.ts (hunk 1 above)
Comment (ashchan) [reply packages/neuron-ui/src/services/remote/apiMethodWrapper.ts]: What did you mean by including the error of that? I moved that `const` of controller not loadable to inner object and return that.

BTW `RemoteNotLoadError` was not used as well - but it's exported.

File: packages/neuron-ui/src/services/remote/apiMethodWrapper.ts (hunk 3 above)
Comment (ashchan) [reply packages/neuron-ui/src/services/remote/apiMethodWrapper.ts]: Updated w/ 18e7a327.

**Pull request merged**